README.md

```diff
@@ -1,7 +1,16 @@
+[content-schemas]: https://github.com/alphagov/govuk-content-schemas
+
 # Content Publisher
 
 A unified publishing application for content on GOV.UK
 
+## Nomenclature
+
+  * Content - Some text ([and related fields][content-schemas]) a user wants to publish
+  * Revision - A version of a piece of content in a particular locale
+  * Edition - A revision that is in the Publishing API (draft or live stack)
```

Comment:
Should we remove draft? as didn't we talk about pushing each revision to the draft stack which would then mean we'd end up with an edition for each revision.

Then if an edition is a revision which exists in the live stack then we could say then an edition is explicitly created when the revision is published.

Reply:
I think we should keep the reference to draft/live, as we know people already refer to 'draft edition' as a thing. Can you think of a better word to use for the revision that exists in the draft content store?